test/CMakeLists.txt

```diff
@@ -23,25 +23,26 @@ ADD_TEST(roundtrip_tests test_kvtree)
 # print test script
 FIND_PROGRAM(PRINT_TEST ${CMAKE_CURRENT_SOURCE_DIR}/print_test)
 ADD_TEST(NAME print_test COMMAND ${PRINT_TEST} ${PROJECT_BINARY_DIR}/src/kvtree_print ${CMAKE_CURRENT_SOURCE_DIR}/files)
-
-IF(MPI_FOUND)
```

Comment:
We need to keep the MPI found if statement since KVTree does not require MPI (but the bcast test does).